frontend/src/safety.ts

```diff
@@ -0,0 +1,353 @@
+    ) ||
+    summary.metadata.permissions.filesystems?.some(
+      (x) => x.toLowerCase() === "host",
+    ) ||
+    summary.metadata.permissions.filesystems?.some(
+      (x) => x.toLowerCase() === "host:rw",
+    )
+  ) {
+    appSafetyRating = addToSafteyRating(
+      appSafetyRating,
+      SafetyRating.unsafe,
+      "can-read-write-all-your-data",
+    )
+    noPermissions = false
+  }
+
+  // read all your data
+  if (
+    summary.metadata.permissions.filesystems?.some(
+      (x) => x.toLowerCase() === "home:ro",
+    ) ||
+    summary.metadata.permissions.filesystems?.some(
+      (x) => x.toLowerCase() === "~:ro",
+    ) ||
+    summary.metadata.permissions.filesystems?.some(
+      (x) => x.toLowerCase() === "host:ro",
+    )
+  ) {
+    appSafetyRating = addToSafteyRating(
+      appSafetyRating,
+      SafetyRating.unsafe,
+      "can-read-all-your-data",
+    )
+    noPermissions = false
+  }
+
+  // read/write your downloads
+  if (
+    summary.metadata.permissions.filesystems?.some(
+      (x) => x.toLowerCase() === "xdg-download",
+    ) ||
+    summary.metadata.permissions.filesystems?.some(
+      (x) => x.toLowerCase() === "xdg-download:rw",
+    )
+  ) {
+    appSafetyRating = addToSafteyRating(
+      appSafetyRating,
+      SafetyRating.potentially_unsafe,
+      "can-read-write-your-downloads",
+    )
+    noPermissions = false
+  }
+
+  // read your downloads
+  if (
+    summary.metadata.permissions.filesystems?.some(
+      (x) => x.toLowerCase() === "xdg-download:ro",
+    )
+  ) {
+    appSafetyRating = addToSafteyRating(
+      appSafetyRating,
+      SafetyRating.potentially_unsafe,
+      "can-read-your-downloads",
+    )
+    noPermissions = false
+  }
+
+  //   can access arbitrary files
+  if (
+    summary.metadata.permissions.filesystems?.length > 0 &&
+    !summary.metadata.permissions.filesystems?.some(
+      (x) =>
+        x.toLowerCase() === "home" ||
+        x.toLowerCase() === "home:rw" ||
+        x.toLowerCase() === "~" ||
+        x.toLowerCase() === "~:rw" ||
+        x.toLowerCase() === "host" ||
+        x.toLowerCase() === "host:rw" ||
+        x.toLowerCase() === "home:ro" ||
+        x.toLowerCase() === "~:ro" ||
+        x.toLowerCase() === "host:ro" ||
+        x.toLowerCase() === "xdg-download" ||
+        x.toLowerCase() === "xdg-download:rw" ||
+        x.toLowerCase() === "xdg-download:ro",
+    )
+  ) {
+    appSafetyRating = addToSafteyRating(
+      appSafetyRating,
+      SafetyRating.potentially_unsafe,
+      "can-access-arbitrary-files",
+    )
+    noPermissions = false
+  }
+
+  // can access and change user settings
+  if (
+    summary.metadata.permissions["session-bus"]?.talk.some(
+      (x) => x.toLowerCase() === "ca.desrt.dconf",
+    )
+  ) {
+    appSafetyRating = addToSafteyRating(
+      appSafetyRating,
+      SafetyRating.potentially_unsafe,
+      "can-access-and-change-user-settings",
+    )
+    noPermissions = false
+  }
+
```

Comment:
For all not listed dbus access, I think we have to consider it potentially unsafe and list them out.

We could potentially describe all `--own-name=` permissions as ability to impersonate other software?

Reply:
I think we can whitelist `org.mpris.MediaPlayer2.*`